sdk/search/azure-search/pom.xml

```diff
@@ -29,7 +29,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.3.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
+      <version>1.3.0-beta.2</version> <!-- {x-version-update;unreleased_com.azure:azure-core;dependency} -->
```

Comment:
This is azure-search. Don't think this should be updated.

Reply:
The build was failing before this change because it was complaining that a version wasn't updated.

Reply:
This was fixed yesterday with https://github.com/Azure/azure-sdk-for-java/pull/8188. This mismatch was the result of two independency PRs being auto-merged. You'll have to rebase and push another commit without this file.